Add additional instructions to UIScene Migration (#12592)
flutter/website: src/content/release/breaking-changes/uiscenedelegate.md

## Migration guide for Flutter apps

### Auto-Migrate (Experimental)

The Flutter CLI can automatically migrate your app if your AppDelegate has not
been customized.

1. Enable UIScene Migration Feature

```console
flutter config --enable-uiscene-migration
```

2. Build or run your app

```console
flutter run
or
flutter build ios
```

If the migration succeeds, you will see a log that says "Finished migration to
UIScene lifecycle". Otherwise, it will warn you to migrate manually using the

> below instructions. If the migration succeeds, no further action is required!

Reviewer: ```suggestion
UIScene lifecycle". Otherwise, it warns you to migrate manually using the
included instructions. If the migration succeeds, no further action is required!
```

</dict>
```

### Create a SceneDelegate (Optional)

If you need access to the SceneDelegate, you can create your own that

> subclasses the `FlutterSceneDelegate`.

Reviewer: ```suggestion
If you need access to the `SceneDelegate`, you can create one by
subclassing `FlutterSceneDelegate`.
```

1. Open your app in Xcode

> 2. Right click the Runner folder and select "New Empty File"

Reviewer: ```suggestion
2. Right click the **Runner** folder and select **New Empty File**
```
